runtime/cudaq/cudaq.cpp

```diff
@@ -300,27 +289,31 @@ std::string get_quake_by_name(const std::string &kernelName,
   // Find the quake code
   std::optional<std::string> result;
   std::shared_lock<std::shared_mutex> lock(globalRegistryMutex);
-  for (auto [k, v] : quakeRegistry) {
-    if (k == kernelName) {
+
+  for (const auto &pair : quakeRegistry) {
+    if (pair.first == kernelName) {
       // Exact match. Return the code.
-      return v;
+      return pair.second;
     }
-    if (k.starts_with(kernelNamePrefix)) {
+
+    if (pair.first.starts_with(kernelNamePrefix)) {
       // Prefix match. Record it and make sure that it is a unique prefix.
       if (result.has_value()) {
-        if (throwException)
+        if (throwException) {
           throw std::runtime_error("Quake code for '" + kernelName +
                                    "' has multiple matches.\n");
-      } else {
-        result = v;
+        } else {
```

Comment:
This `else` used to be paired with the `if (result.has_value())`, but with the brace change, it looks like it's now paired with `if (throwException)`. Is that intentional?